### PR TITLE
update readme files

### PR DESCRIPTION
**Pull request by KseniyaKuzmina**

### CODE REVIEW

File: CS/AreaSegmentColorizer/Readme.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+# Fill Area Series by Segments
 
 In this example, the area chart visualizes the visible light spectrum.
 
-<img src="./img/area-series-segment-colorizer.png" width="85%" />
+<img src="./img/area-series-segment-colorizer.png" width="85%" alt="Area series with custom fill"/>
+
+Follow the steps below to show such a chart:
+
+* Add an [Area Series](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.AreaSeries) object to the [ChartView.Series](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.ChartView.Series) collection, and set its [Data](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.XYSeries.Data) property to bind the series to a data source.
```
Comment (dirk-pieterse) [inline CS/AreaSegmentColorizer/Readme.md]: `and set its [Data](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.XYSeries.Data) property` Set it to what?

File: CS/AreaSegmentColorizer/Readme.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+
+* Set the series' [FillColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.AreaSeries.FillColorizer) property to a [SegmentBasedFillColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.SegmentBasedFillColorizer) object, and configure the colorizer options to define area fill settings.
```
Comment (dirk-pieterse) [inline CS/AreaSegmentColorizer/Readme.md]:
```suggestion
* Set the series' [FillColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.AreaSeries.FillColorizer) property to a [SegmentBasedFillColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.SegmentBasedFillColorizer) object and configure the colorizer options to define area fill settings.
```

File: CS/CustomPointColorizer/Readme.md [hunk 3]
```diff
@@ -1,23 +1,55 @@
 # Color Points by Their Arguments
 
-In this example, the bar chart displays monthly values colored based on the season. For a complete description, refer to the following help topic: [Series Point Colorizers](https://docs.devexpress.com/MAUI/403339/charts/series-point-colorizers).
+In this example, the bar chart displays monthly values colored based on the season. For a complete description, refer to the following help topic: [Series Point Colorizers](https://docs.devexpress.com/MAUI/403339/charts/series-point-colorizers). 
+
+![Apply a custom coloring rule to bars](./img/chart-custom-point.png)
+
+To paint bars by seasons, assign a [custom colorizer](https://docs.devexpress.com/MAUI/403339/charts/series-point-colorizers#custom-point-colorizers) to a [bar series](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BarSeries)' [PointColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BarSeries.PointColorizer) property. In this example, the custom colorizer selects a color depending on the month serial number.
```
Comment (dirk-pieterse) [inline CS/CustomPointColorizer/Readme.md]:
```suggestion
To paint bars according to the season, assign a [custom colorizer](https://docs.devexpress.com/MAUI/403339/charts/series-point-colorizers#custom-point-colorizers) to a [bar series](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BarSeries)' [PointColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BarSeries.PointColorizer) property. In this example, the custom colorizer selects a color depending on the month's serial number.
```

File: CS/CustomValueRangeColorizer/Readme.md [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
 
-Please register the DevExpress NuGet Gallery in Visual Studio to restore the NuGet packages used in this solution. See the following topic for more information: [Get Started with DevExpress Mobile UI for .NET MAUI](https://docs.devexpress.com/MAUI/403249/get-started).
+* **Country** - Stores data for bubble arguments shown on the x-axis.
+* **Gdp** - Stores data for bubble values shown on the y-axis.
+* **Population** - Stores data for bubble weight values that define the bubble size.
+* **Hpi** - Stores values that a colorizer uses to determine the bubble color.
 
-You can also refer to the following YouTube video for more information on how to get started with the DevExpress .NET MAUI Controls: [Setting up a .NET MAUI Project](https://www.youtube.com/watch?v=juJvl5UicIQ).
+To paint bubbles by the ranges of underlying values, assign a [CustomValueBandPointColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.CustomValueBandPointColorizer) to a [BubbleSeries](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BubbleSeries)' [PointColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BubbleSeries.PointColorizer) property.
```
Comment (dirk-pieterse) [inline CS/CustomValueRangeColorizer/Readme.md]:
```suggestion
To paint bubbles according to the ranges of underlying values, assign a [CustomValueBandPointColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.CustomValueBandPointColorizer) to a [BubbleSeries](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BubbleSeries)' [PointColorizer](https://docs.devexpress.com/MAUI/DevExpress.Maui.Charts.BubbleSeries.PointColorizer) property. 
```

**Review by dirk-pieterse — APPROVED** [review]

File: CS/AreaSegmentColorizer/Readme.md (hunk 1 above)
Comment (KseniyaKuzmina) [reply CS/AreaSegmentColorizer/Readme.md]: I'll replace 'set' with 'specify'.

**Pull request merged**